docs/src/main/sphinx/connector/druid.rst

```diff
@@ -57,6 +57,49 @@ name from the properties file.
 Type mapping
 ------------
 
+Because Trino and Druid each support types that the other does not, this
+connector modifies some types when reading data. Refer to
+the following table for type mapping.
```

Comment:
Remove "Refer to the following table for type mapping", it seems overkill when there's only one table (which is introduced below anyway)

Reply:
Done